lunatrace/bsl/backend/src/graphql-yoga/resolvers/create-pull-request-for-vulnerability.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Business Source License v1.1
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * https://github.com/lunasec-io/lunasec/blob/master/licenses/BSL-LunaTrace.txt
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { GraphQLYogaError } from '@graphql-yoga/node';
+import {
+  PullRequestOctokit,
+  replacePackageAndFileGitHubPullRequest,
+} from '@lunatrace/npm-package-cli/src/package/github-pr';
+
+import { getInstallationAccessToken } from '../../github/auth';
+import { hasura } from '../../hasura-api';
+import { log } from '../../utils/log';
+import { MutationResolvers } from '../generated-resolver-types';
+import { checkProjectIsAuthorized, isAuthenticated } from '../helpers/auth-helpers';
+
+type CreatePullRequestForVulnerabilityType = NonNullable<MutationResolvers['createPullRequestForVulnerability']>;
+
+function splitGitHubRepoPath(gitHubRepo: string) {
+  const split = gitHubRepo.split('/');
```

Comment:
https://nodejs.org/api/path.html#pathsep

Reply:
doesn't apply here since this is a URL returned from GitHub (`git://github.com/org/repo.git`)